app/views/spree/admin/shared/_edit_resource_links.html.haml

```diff
@@ -1,4 +1,4 @@
 .form-buttons.filter-actions.actions
-  = button t('update'), 'icon-refresh'
+  = button t(:update), 'icon-refresh'
```

Comment:
This is semantically the same, right? Just better way of writing it.

Reply:
here we move from the key spree.actions.update to the key update.

Reply:
Really? How does `t('update')` know to look in spree.actions.update? Or do you mean in combination with other commits?

Reply:
🙈 sorry Maikel, I saw a inexistent dot. yes, this is the same but better looking :-) and less  confusion with the lazylookup.